### PR TITLE
doc: cleaning up the readme.md

### PR DESCRIPTION
**Pull request by peknis**
Cleaning up the language and formatting in the
README.md document file.

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

**Comment by peknis** [comment]
Not ready for review yet. Need to first figure out whether we can use the abbreviation NCS. @carlescufi seems that I cannot add reviewers. Am I missing some rights or something?

**Comment by carlescufi** [comment]
> Not ready for review yet. Need to first figure out whether we can use the abbreviation NCS. @carlescufi seems that I cannot add reviewers. Am I missing some rights or something?

We cannot use the abbreviation because this is not rendered by Sphinx, it's just an `.md` file rendered by GitHub itself. The guide can be found here:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Try again to add reviewers, should work now

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# nRF Connect SDK Example Application
+# nRF Connect SDK Example application
```
Comment (carlescufi) [inline README.md:1]: why keep the upper case in "Example" then?

**Comment by carlescufi** [comment]
Thanks for the PR @peknis! This is great stuff.

File: README.md (hunk 1 above)
Comment (peknis) [reply README.md:1]: So true. I was first thinking of it as a name for the application, thus capitalized Example. Now, thinking again and I think you are right. Let's have it all lower case.

**Review by gmarull — COMMENTED** [review]
we should upstream changes first

**Comment by peknis** [comment]
> we should upstream changes first

Are you expecting any actions from tech pubs?

**Comment by gmarull** [comment]
> > we should upstream changes first
> 
> Are you expecting any actions from tech pubs?

maybe open a PR here: https://github.com/zephyrproject-rtos/example-application (we'll sync this fork periodically, so any generic changes should go there first)

**Comment by peknis** [comment]
Here goes my PR to the Zephyr example application https://github.com/zephyrproject-rtos/example-application/pull/29

**Comment by gmarull** [comment]
shouldn't we pull upstream changes, revert previous noup and re-apply it with changes?

**Comment by carlescufi** [comment]
> shouldn't we pull upstream changes, revert previous noup and re-apply it with changes?

Hmm. The `main` branch already has the upstream changes. @peknis did you rebase this PR against the current `main`?

**Comment by carlescufi** [comment]
Ah ok I see what you mean now @gmarull.
@peknis, can you please send all the changes that are not NCS-specific upstream first?

**Comment by gmarull** [comment]
> Ah ok I see what you mean now @gmarull, @peknis added some additional changes that should go upstream instead.
> 
> @peknis, can you please send all the changes that are not NCS-specific upstream first?

this was already done, right? I mean to keep a single `noup`.

**Comment by peknis** [comment]
> > Ah ok I see what you mean now @gmarull, @peknis added some additional changes that should go upstream instead.
> > @peknis, can you please send all the changes that are not NCS-specific upstream first?
> 
> this was already done, right? I mean to keep a single `noup`.

I believe I have done everything needed already.

**Comment by carlescufi** [comment]
> > Ah ok I see what you mean now @gmarull, @peknis added some additional changes that should go upstream instead.
> > @peknis, can you please send all the changes that are not NCS-specific upstream first?
> 
> this was already done, right? I mean to keep a single `noup`.

It was done but @peknis has added more changes in this PR that are not NCS-specific.
After all is done I will rebase to squash @peknis' commit into the single `noup`.

**Review by gmarull — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**